### ISSUE TITLE
Phone Number Error

### ISSUE THREAD

**Issue by Salman98** [Bruno-Furtado/react-native-sms-retriever#2]
### Environment

* React Native Version
* Platform:  Android  
* Device:  Samsung
* OS Version 7
* react-native-sms-retriever version@1.0.1
* Build Tools Version (Android only)

### Description

i have added this lib and run your sample code but i received an error 
{"framesToPop":1,"code":"ACTIVITY_RESULT_NOOK_ERROR_TYPE"}

**Comment by Bruno-Furtado** [comment]
@Salman98,

This is an error handling for the cases where activity result is not ok.
Usually this error occurs when you do not select a hint picket number.


Maybe there some other problem...
Can you debug [this method](https://github.com/Bruno-Furtado/react-native-sms-retriever/blob/1.0.1/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java#L153) and share the results here?

**Comment by Bruno-Furtado** [comment]
@Salman98, any other questions?

**Comment by Salman98** [comment]
> @Salman98, any other questions?

yes thanks for asking. I tried alot but i am unable to fix this , then i tried your example but it is also not working . Can you please tell me the complete steps

**Comment by Bruno-Furtado** [comment]
@Salman98, can you debug this method and share the results here?

**Comment by Salman98** [comment]
In this class "PhoneNumberHelper"
![image](https://user-images.githubusercontent.com/14360952/52410016-f019f600-2af8-11e9-80bb-132469ade855.png)
 After debugging the function i am getting following responses:

REQUEST_PHONE_NUMBER_REQUEST_CODE = 1
requestCode= 1
resultCode = 1002
Activity.RESULT_OK = -1

**Comment by Bruno-Furtado** [comment]
The HintRequest API provided by Google has not fully accomplished its feature and is quite buggy.
See more details [here](https://stackoverflow.com/questions/46900310/play-services-hint-request-cannot-display-phone-numbers-when-requested/53607016#53607016).

**Comment by Bruno-Furtado** [comment]
@Salman98, check [this comment](https://github.com/Bruno-Furtado/react-native-sms-retriever/issues/3#issuecomment-471206532) as well.
